build/schemas/cal/date-time.json

```diff
@@ -0,0 +1,14 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://gobl.org/draft-0/cal/date-time",
+  "$ref": "#/$defs/DateTime",
+  "$defs": {
+    "DateTime": {
+      "type": "string",
+      "format": "date-time",
+      "title": "Date Time",
+      "description": "Civil date time in simplified ISO format with no time zone\ninformation, for example: 2021-05-26T13:45:00\n"
```

Comment:
Do we want those line breaks in that description?

Reply:
I fixed this too. Any empty lines at the end of the here.Doc are deleted.

Reply:
I also meant the line breaks in the middle. Those line breaks are there originally to force text wrapping in a monospaced presentation (i.e. a code editor), but this is the JSON schema and that description could be used, for example, to automatically generate HTML documentation and we won't want a `<br>` or a `<p>` to be added at those places.

Reply:
There isn't a great solution for that... Markdown will of course ignore single `\n` and they are not meaningful inside HTML, so I think they're safe to ignore.

Reply:
It doesn't feel 100% safe to assume those `\n` will always be ignored. Arguably, if those line breaks are expected to be ignored, then they shouldn't be there in the first place. We could write our own helper to remove new lines unless there are more than one consecutively.

Reply:
While not the case yet, imagine if we had code samples inside the `here.Doc()`. You for sure don't want to remove the `\n` and I'm not sure what rules could be used to do that.

Reply:
ok then